examples/defaulter-gen/generators/defaulter.go

```diff
@@ -401,6 +411,18 @@ func newCallTreeForType(existingDefaulters, newDefaulters defaulterFuncMap) *cal
 	}
 }
 
+func populateDefaultValue(node *callNode, commentLines []string) *callNode {
+	// TODO: Check eligibility of defaulting
```

Comment:
Options here: 1. replicate what @apelisse did in his OpenAPI PR here, 2. do nothing let the OpenAPI code own the eligibility

WDYT @apelisse ?

Reply:
I'd go with 1., we can't hope that the other system catches the problem, that'd be an implicit dependency between the two.